### PR TITLE
Star raku

### PR DESCRIPTION
**Pull request by stoned**
Complete what was already done with respect to $*RAKU. Ref #3187

### CODE REVIEW

File: doc/Language/variables.pod6 [hunk 1]
```diff
@@ -1532,6 +1532,9 @@ It stringifies to C<Raku>:
 
     $*RAKU.put; # OUTPUT: «Raku␤»
 
+B<Note:> Before Rakudo version 2020.1, this information was available
+through the C<$*PERL> variable.
+
 X<|$*PERL>
 =head4 C<$*PERL>
```
Comment (JJ) [inline doc/Language/variables.pod6]: It still is. We cam maybe discourage its use, but it's still there.

File: doc/Type/Compiler.pod6 [hunk 2]
```diff
@@ -31,7 +34,8 @@ It's empty, but it might contain the codename for specific releases.
 
     method build-date()
 
-Up to version 2019.03.1, it returns the date when it was built.
+Up to Rakudo version 2019.03.1, hence only with C<$*PERL.compiler>,
```
Comment (JJ) [inline doc/Type/Compiler.pod6]: Don't really understand this sentence. The hence, mainly.

File: doc/Type/Compiler.pod6 [hunk 3]
```diff
@@ -31,7 +34,8 @@ It's empty, but it might contain the codename for specific releases.
...
+it returns the date when it was built.
```
Comment (JJ) [inline doc/Type/Compiler.pod6]: *returned*?

Comment (stoned) [reply doc/Type/Compiler.pod6]: indeed!

File: doc/Type/Compiler.pod6 (hunk 2 above)
Comment (stoned) [reply doc/Type/Compiler.pod6]: yes it was a bit cryptic. I simply removed this :-)

File: doc/Language/variables.pod6 (hunk 1 above)
Comment (stoned) [reply doc/Language/variables.pod6]: I mentioned both are now available, for the time being.

**Pull request merged**